### PR TITLE
Aly

### PR DESCRIPTION
**Pull request by alypriolo**
hi caleb! please accept my friend request.

### CODE REVIEW

File: Navigation/MainStackNav.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ import { createStackNavigator, StackView } from '@react-navigation/stack';
 import * as React from 'react';
 import LoginPage from '../Pages/Login.js';
 import HomePage from '../Pages/Home.js';
+import ChildHomePage from '../Pages/ChildHomePage.js';
```
Comment (lunstb) [inline Navigation/MainStackNav.js:6]: Reformat top level directory

File: Pages/ChildSaveBank.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    BackButton: {
```
Comment (lunstb) [inline Pages/ChildSaveBank.js:39]: consistent naming convention on capitalization.

File: Pages/ChildHomePage.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
+        alignItems: 'center',
+        justifyContent: 'center',
```
Comment (lunstb) [inline Pages/ChildHomePage.js:37]: See below

File: Pages/ChildSaveBank.js (hunk 2 above)
Comment (lunstb) [reply Pages/ChildSaveBank.js:39]: Also, this looks like the same code as mentioned above. Try to have a theme that is used in multiple places

File: Pages/ChildShareBank.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        <Text>Tooth fairy money +$4.00</Text>
+        <Text>Grandma gave me money for being nice +$2.00</Text>
+        <Text>Mom took away money for cursing -$5.00</Text>
+    
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
```
Comment (lunstb) [inline Pages/ChildShareBank.js:33]: See above

File: Pages/ChildSaveBank.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            style={styles.BackButton}
+            title="<--"
+            onPress={() => navigation.goBack()}
+        />
+        <Text>Save Bank:</Text>
+      
+        <Button 
+            style={styles.NewSaveTransaction}
+            title="New Transaction"
+            onPress={() => navigation.navigate('EnterSaveTransaction')}
```
Comment (lunstb) [inline Pages/ChildSaveBank.js:21]: Best coding practice, put this in an object. https://stackoverflow.com/questions/4286899/javascript-enum-to-corresponding-string-value. Makes it easier to debug

File: Pages/ChildSpendBank.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <Text>Past Transactions:</Text>
+        <Text>Tooth fairy money +$4.00</Text>
+        <Text>Grandma gave me money for being nice +$2.00</Text>
+        <Text>Mom took away money for cursing -$5.00</Text>
+    
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
```
Comment (lunstb) [inline Pages/ChildSpendBank.js:32]: Same thing with the style sheets

File: Pages/ChildSpendBank.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            style={styles.BackButton}
+            title="<--"
+            onPress={() => navigation.goBack()}
+        />
+        <Text>Spend Bank:</Text>
+      
+        <Button 
+            style={styles.NewSpendTransaction}
+            title="New Transaction"
+            onPress={() => navigation.navigate('EnterSpendTransaction')}
```
Comment (lunstb) [inline Pages/ChildSpendBank.js:21]: Same enum thing

File: Pages/EnterSaveTransaction.js [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+            title="Withdraw Money"
+            //onPress={() => navigation.navigate('EnterShareTransaction')}
+            //i think this should not pop up to a new page but rather a notification
+            //kinda like venmo how it goes chaching for deposit and vibrate for withraw
+        />
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
```
Comment (lunstb) [inline Pages/EnterSaveTransaction.js:44]: same style thing

Comment (lunstb) [reply Pages/EnterSaveTransaction.js:44]: Just go over all of your style sheets and keep track of which things are the same. Take all of them out and put them in a theme class or something. DRY (Don't Repeat Yourself)

File: Pages/EnterShareTransaction.js [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+            title="Withdraw Money"
+            //onPress={() => navigation.navigate('EnterShareTransaction')}
+            //i think this should not pop up to a new page but rather a notification
+            //kinda like venmo how it goes chaching for deposit and vibrate for withraw
+        />
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
```
Comment (lunstb) [inline Pages/EnterShareTransaction.js:44]: A note on this is that not EVERYTHING should be in the theme css stuff. A lot of the stuff here is probably specific to this page and should be left here bc if you clutter the main theme it gets messy.

**Comment by lunstb** [comment]
Also! Brief note on naming conventions, probably name the PR based on what the PR does. Additionally, try to name your commits effectively too

File: Navigation/MainStackNav.js (hunk 1 above)
Comment (alypriolo) [reply Navigation/MainStackNav.js:6]: can u specify further how you would like the directory to be reorganized? it's pretty comparable to the survivorFitness directory so def could use some clarification on how to improve it. thx!

**Pull request merged**